lib/job.ml

```diff
@@ -71,7 +71,7 @@ let log_path job_id =
       | Ok false -> Error (`Msg (Fmt.str "Job log %a does not exist" Fpath.pp path))
       | Error _ as e -> e
     end
-  | _ -> Error (`Msg (Fmt.str "Invalid job ID %S" job_id))
+  | _ -> Error (`Msg (Fmt.str "Invalid job ID \"%s\"" job_id))
```

Comment:
Is this change intentional?

Reply:
I took to literally your "I'm not keen on removing the quotes". Removed the commit.